concrete/src/Gathering/Item/Template/Template.php

```diff
@@ -196,7 +194,7 @@ public function getGatheringItemTemplateSlotWidth(GatheringItem $item)
 
         $w = 0;
         $handles = $this->getGatheringItemTemplateFeatureHandles();
-        $assignments = \Concrete\Core\Feature\Assignment\GatheringItemAssignment::getList($item);
+        $assignments = GatheringItemAssignment::getList($item);
```

Comment:
Did `GatheringItemAssignment` move? It looks like it won't properly resolve since the file was deleted and `use` was removed

Reply:
Yes, I'm going to fully strip these files out in another pull request. The feature and functionality are going to be fully removed from the core.